protobuf-maven-plugin/src/it/protoc-descriptor-file/selector.groovy

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2023 - 2024, Ashley Scopes.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Only execute this test if the user has protoc installed on their
+// PATH and it is successfully executable.
+try {
```

Comment:
Do we need this check, or can we use the Maven-provided protoc? (e.g. check the java-main invoker tests)

Think we can just remove this check unless there is a reason to prefer using a $PATH based protoc for this test case 😄